airbyte/metadata.csv

```diff
@@ -0,0 +1,52 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
+airbyte.cron.jobs_run,count,,,,Number of cron runs by cron type.,0,airbyte,cron jobs run,
+airbyte.cron.workflows_healed,count,,,,Number of workflow the self healing cron healed.,0,airbyte,workflows healed,
+airbyte.metrics_reporter.est_num_metrics_emitted_by_reporter,count,,,,Estimated metrics emitted by the reporter in the last interval. This is estimated since the count is not precise.,0,airbyte,est num metrics emitted by reporter,
+airbyte.metrics_reporter.num_active_conn_per_workspace,distribution,,,,Number of active connections per workspace.,0,airbyte,num active conn per workspace,
+airbyte.metrics_reporter.num_orphan_running_jobs,gauge,,job,,Number of jobs reported as running that as associated to connection inactive or deprecated.,0,airbyte,num orphan running jobs,
+airbyte.metrics_reporter.num_pending_jobs,gauge,,job,,Number of pending jobs.,0,airbyte,num pending jobs,
+airbyte.metrics_reporter.num_running_jobs,gauge,,job,,Number of running jobs.,0,airbyte,num running jobs,
+airbyte.metrics_reporter.num_total_scheduled_syncs_last_day,gauge,,,,Number of total syncs runs in last day.,0,airbyte,num total scheduled syncs last day,
+airbyte.metrics_reporter.num_unusually_long_syncs,gauge,,,,Number of unusual long syncs compared to their historic performance.,0,airbyte,num unusually long syncs,
+airbyte.metrics_reporter.oldest_pending_job_age_secs,gauge,,second,,Oldest pending job in seconds.,0,airbyte,oldest pending job age secs,
+airbyte.metrics_reporter.oldest_running_job_age_secs,gauge,,second,,Oldest running job in seconds.,0,airbyte,oldest running job age secs,
+airbyte.metrics_reporter.overall_job_runtime_in_last_hour_by_terminal_state_secs,distribution,,second,,Overall job runtime - scheduling and execution for all attempts - for jobs that reach terminal states in the last hour. Tagged by terminal states.,0,airbyte,overall job runtime in last hour by terminal state secs,
+airbyte.orchestrator.source_hearbeat_failure,count,,,,Fail a replication because the source missed an heartbeat.,0,airbyte,source hearbeat failure,
+airbyte.server.breaking_change_detected,count,,,,A breaking schema change has been detected.,0,airbyte,breaking change detected,
```

Comment:
Reading the description I thought it was a boolean and not the total number of breaking changes